ocs_ci/ocs/resources/pod.py

```diff
@@ -143,6 +145,59 @@ def exec_ceph_cmd(self, ceph_cmd, format='json-pretty'):
             return [item for item in out if item]
         return out
 
+    def run_io(
```

Comment:
would be better to call this function "run_fio()" as we'll have other ways of running IO with different workloads

Reply:
but this is THE pod method to run IO so I think we are good.
All other functions like dd should be removed